File: FormDetails.java
import java.util.*;

public class FormDetails{
	public String filename;
	public boolean isTemplate;
	public List<Question> questions;

	public FormDetails(String file, boolean t){
		filename = file;
		isTemplate = t;
		questions = new ArrayList<Question>;
	}
	public FormDetails(){
		filename = "UnknownFile";
		isTemplate = false;
		questions = new ArrayList<Question>;
	}

	public void addQ(Question q){
		questions.add(q);
	}
}

File: back_end/FormProcessorApi/FormProcessorApi/Models/FormDetails.cs
﻿using Microsoft.AspNetCore.Http;
using System.ComponentModel.DataAnnotations.Schema;

namespace FormProcessorApi.Models
{
    public class FormDetails
    {
        public int FormDetailsId { get; set; }
        public int TemplateId { get; set; }
        public string Filename { get; set; }
        public bool IsTemplate { get; set; }

        public Question[] Questions { get; set; }

        [NotMapped]
        public IFormFile FormImage { get; set; }
    }
}
